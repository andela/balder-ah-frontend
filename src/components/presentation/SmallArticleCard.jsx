import React from 'react';

const SmallArticleCard = () => (
  <div className="col s12 m6">
    <div className="card horizontal hoverable">
      <div className="card-image">
        <img src="https://lorempixel.com/100/190/nature/6" alt="smith" />
      </div>
      <div className="card-stacked">
        <div className="card-content">
          <p>I am a very simple card. I am good at containing small bits of information.</p>
        </div>
        <div className="card-action">
          <a href="#!">Read Now</a>
        </div>
      </div>
    </div>
  </div>
);

export default SmallArticleCard;
