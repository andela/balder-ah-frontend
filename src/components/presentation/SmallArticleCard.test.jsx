import React from 'react';
import { render, cleanup } from 'react-testing-library';
import 'jest-dom/extend-expect';
import SmallArticleCard from './SmallArticleCard';

afterEach(cleanup);

describe('<SmallArticleCard />', () => {
  test('should render correctly', () => {
    const { container } = render(<SmallArticleCard />);
    expect(container).toMatchSnapshot();
  });
});
