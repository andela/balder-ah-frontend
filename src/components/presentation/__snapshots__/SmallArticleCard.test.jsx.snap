// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SmallArticleCard /> should render correctly 1`] = `
<div>
  <div
    class="col s12 m6"
  >
    <div
      class="card horizontal hoverable"
    >
      <div
        class="card-image"
      >
        <img
          alt="smith"
          src="https://lorempixel.com/100/190/nature/6"
        />
      </div>
      <div
        class="card-stacked"
      >
        <div
          class="card-content"
        >
          <p>
            I am a very simple card. I am good at containing small bits of information.
          </p>
        </div>
        <div
          class="card-action"
        >
          <a
            href="#!"
          >
            Read Now
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
